config/logging.yaml

```diff
@@ -49,15 +59,34 @@
 #     [%X{session}] session ID from cookies, else the request-ID
 #     [%X{session_safe}] random ID assigned per session, not available early on during
 #                        request logging, but prevents disclosure of the session ID
-#     [%X{string}] variable set using ::Logging.mdc['string'] =
-
+#     [%X{string}] variable set via MDC (::Logging.mdc['string'])
+#     [%x{string}] variable set via NDC (::Logging.ndc['string'])
+#
+# :sys_pattern:
+#   Logger line pattern for system log/journal. See :pattern for more info.
+#
+# :facility:
+#   System log/journal facility to use. One of LOG_AUTH, LOG_AUTHPRIV, LOG_CONSOLE,
+#   LOG_CRON, LOG_DAEMON, LOG_FTP, LOG_KERN, LOG_LPR, LOG_MAIL, LOG_NEWS, LOG_NTP,
+#   LOG_SECURITY, LOG_SYSLOG, LOG_USER, LOG_UUCP, LOG_LOCAL0 to LOG_LOCAL7.
```

Comment:
Is there a good resource to look up what these facilities mean? If yes, should we consider putting a link here in the comments?

Reply:
Sure, adding this to the text. It's in `man 3 syslog`.